src/stylesheets/components/_header.scss

```diff
@@ -116,8 +117,12 @@
       box-shadow: 0 0 0 govuk-colour("blue");
     }
 
+    .js-enabled & {
+      display: inline;
+    }
+
     @include govuk-media-query($from: tablet) {
-      display: none;
+      display: none !important;
```

Comment:
Rather than needing to use `!important` here this could we flip this media query on its head and do e.g.

```scss
.js-enabled & {
  @include govuk-media-query($from: tablet) {
    display: inline;
  }
}
```

?

Reply:
I'll give it a go.